mods/cnc/maps/nod10b/rules.yaml

```diff
@@ -2,7 +2,7 @@ World:
 	LuaScript:
 		Scripts: campaign.lua, utils.lua, nod10b.lua
 	MissionData:
-		Briefing: GDI is developing a new, heavily armored tank. Our spies have located the GDI R&D base.\n\nUse your small strike force to locate and penetrate the center. Be sure to destroy the tanks in the base and the factory.
+		Briefing: GDI is developing a new, heavily armored tank. Our spies have located the GDI R&D base.\n\nUse your small strike force to breach the facility. Ensure both the tanks and the factory are destroyed.
```

Comment:
This wording suggests that there are two tanks to be destroyed. Keep previous wording or adjust to remove the "both".